src/main/java/org/opentripplanner/standalone/config/updaters/sources/BikeRentalSourceFactory.java

```diff
@@ -58,9 +58,24 @@ public static BikeRentalDataSourceParameters create(String typeKey, NodeAdapter
 
   public BikeRentalDataSourceParameters create() {
     switch (type) {
-      case GBFS: return new GbfsBikeRentalDataSourceParameters(url(), network(), routeAsCar(), allowKeepingBicycleRentalsAtDestination());
-      case KML:  return new GenericKmlBikeRentalDataSourceParameters(url(), namePrefix());
-      default:   return new BikeRentalDataSourceParameters(type, url(), network(), apiKey());
+      case GBFS:
+        return new GbfsBikeRentalDataSourceParameters(
+            url(),
+            network(),
+            routeAsCar(),
+            allowKeepingBicycleRentalsAtDestination(),
+            c.asMap("headers", NodeAdapter::asText)
```

Comment:
All the other parameters have nice little methods here, it would be nice to follow the same pattern - giving the parameter a name. I think the parameter should be named `httpHeader` since the context is a BikeRentalDataSource.